docker/data/clickhouse/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
No, please just return it back. `.getkeep` does his job pretty well and we have to ignore the whole `docker/data`